packages/design-system/src/components/locale/en.json

```diff
@@ -69,6 +69,16 @@
     "usaLogo": "USA.gov - Government Made Easy",
     "warning": "Warning"
   },
+  "idleTimeoutDialog": {
+    "heading": "Are you still there?",
+    "messageLine1": "You've been inactive for a while.",
+    "messageLine2": "Your session will end in ",
```

Comment:
These could possibly have more descriptive keys like `inactiveMessage` and `sessionWillEndIn`